lesson5/hometask_TetyanaKarpova.rb

```diff
@@ -0,0 +1,22 @@
+class LazyHash < Hash
+  def lazy?(method_name)
+    fetch(method_name).is_a? Proc
+  end
+
+  def []=(key, value)
+    unless self.has_key? key
```

Comment:
A good solution again! But as in the previous case, could be improved.
What If I would like to create a LazyHash with data initially? Like, `LazyHash[:a, 1, :b, 2]`
Obvious, that my LazyHash wouldn't know about methods `#a` and `#b` due to your implementation.